src/nvme/mi-mctp.c

```diff
@@ -84,10 +85,16 @@ struct nvme_mi_transport_mctp {
 	int	sd;
 };
 
+static int tag_ioctl(int sd, unsigned long req, struct mctp_ioc_tag_ctl *ctl)
```

Comment:
Maybe `ioctl_tag` in case you want to add more `ioctls` then all of them have the same prefix?

Reply:
I'm happy with either. We shouldn't have any new forms of `ioctl()` there, as that's all the kernel interface provides. If we do need to extend this later, it's an entirely-internal API, so we can just update all at once.

Reply:
On second thought, the `ioctl_tag` probably makes more sense as we're also prescribing the prototype of the ioctl call there. I'll update this in v2.